hawkbit-repository/hawkbit-repository-api/src/main/java/org/eclipse/hawkbit/repository/model/TargetFilterQuery.java

```diff
@@ -70,4 +70,10 @@
      */
     ActionType getAutoAssignActionType();
 
+    /**
+     * @return the weight of the {@link Action}s created during an auto
+     *         assignment.
+     */
+    Integer getAutoAssignWeight();
```

Comment:
on Rollout, we have `Optional<Integer> getWeight();` - would it make sense to align this?

Reply:
Yes, changed the return type